tests/integ/test_git.py

```diff
@@ -59,15 +62,16 @@ def test_git_support_with_pytorch(sagemaker_local_session):
             predictor.delete_endpoint()
 
 
-def test_git_support_with_mxnet(sagemaker_local_session, mxnet_full_version):
+def test_git_support_with_mxnet(sagemaker_local_session):
     script_path = "mnist.py"
```

Comment:
so if you check the script under tests/data/mxnet_mnist/mnist.py (which test_mxnet_train.py::test_deploy_model and your test both use), you can see that the script is only doing training and saving the model. Your case is different, you are testing the serving script, that is, with a trained model, you serving entry point is an inference script, for example, tests/data/mnxet_mnist/mnist_hosting_with_custom_handlers.py

It’s a bit confusing since with model class, the entry point can either be train.py or inference.py, but the portion of your test is testing whether an inference script from git repo can work.